cmd/soroban-rpc/internal/methods/get_ledger_entries.go

```diff
@@ -64,6 +66,14 @@ func NewGetLedgerEntriesHandler(logger *log.Entry, ledgerEntryReader db.LedgerEn
 					Message: fmt.Sprintf("cannot unmarshal key value %s at index %d", requestKey, i),
 				}
 			}
+			if ledgerKey.Type == xdr.LedgerEntryTypeExpiration {
```

Comment:
is there a test case to assert this error outcome for request of expiration entry key, it looks like `simulate_transaction_test.go` hits getLedgerEntry/Entries and verifies ExpirationLedger values, but didn't see attempts at this path.

Reply:
Ahh.. you're asking if there is a unit test that verifies that expiration ledger keys cannot be requested. Hmm.. no.
We had few such tests that were "fixed" to avoid the error, but I wasn't considering that we should keep these as false positive tests.
I'll add one of these back, just for ensuring complete coverage.